.github/workflows/python-publish.yml

```diff
@@ -28,8 +28,8 @@ jobs:
           python-version: '3.x'
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install build
+          python -m pip install uv
+          uv pip install build
```

Comment:
I'm not sure this will work due to the venv. Since it's just a single package it's ok to leave it with pip. You could also use setup-uv and `uvx build` instead